vsm

```diff
@@ -118,7 +147,7 @@ class State(object):
 
 
 def emit(signal, value):
-    print('{} = {}'.format(signal, value))
+    Log.i('{} = {}'.format(signal, value))
```

Comment:
The change looks good to me now, but you still need to adapt tests.py so that the tests can deal with the timestamp. I think you can just post-process the output to just remove the timestamps before comparing, how does that sound?